.changeset/curly-pandas-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nextui-org/input": major
```

Comment:
@mrbadri 

```suggestion
"@nextui-org/input": patch
```

Update the changeset to be a `patch`, when creating it just press enter without selecting that it is a `major` / `minor`